sycl/doc/extensions/DeviceGlobal/cl_intel_global_variable_access.asciidoc

```diff
@@ -0,0 +1,265 @@
+= cl_intel_global_variable_access
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+== Name Strings
+
+`cl_intel_global_variable_access`
+
+== Contact
+
+Ben Ashbaugh, Intel (ben 'dot' ashbaugh 'at' intel 'dot' com)
+
+== Contributors
+
+// spell-checker: disable
+* Ben Ashbaugh, Intel
+* Joe Garvey, Intel
+* Greg Lueck, Intel
+* Artem Radzikhovskyy, Intel
+// spell-checker: enable
+
+== Notice
+
+Copyright (c) 2021 Intel Corporation. All rights reserved.
+
+== Status
+
+Working Draft
+
+This is a preview extension specification, intended to provide early access to
+a feature for review and community feedback.  When the feature matures, this
+specification may be released as a formal extension.
+
+Because the interfaces defined by this specification are not final and are
+subject to change they are not intended to be used by shipping software
+products.  If you are interested in using this feature in your software
+product, please let us know!
+
+== Version
+
+Built On: {docdate} +
+Revision: A
+
+== Dependencies
+
+This extension is written against the OpenCL API Specification Version 3.0.10.
+
+== Overview
+
+This extension allows host code to read or write data contained within a global
+variable (aka program scope variables) that is defined in a program object.
+
+== New API Functions
+
+[source]
+----
+cl_int clEnqueueReadGlobalVariableINTEL(
+    cl_command_queue command_queue,
+    cl_program program,
+    const char* name,
```

Comment:
Agreed.  This isn't strictly required, but it would be nice to be compatible with the other "read" APIs.

We should also add a "cl_bool blocking_write" to clEnqueueWriteGlobalVariableINTEL.

Reply:
Done in 05d1c30f0b9247c728be8a50ba6c7a2c57e94628.